Products/ZenUI3/browser/javascript.py

```diff
@@ -251,13 +251,19 @@ def snippet(self):
         # 3. the timezone of the browser
         user = self.context.dmd.ZenUsers.getUserSettings()
         timezone = user.timezone
+        date_fmt = user.dateFormat
         snippet = """
+          (function(){
+            Ext.namespace('Zenoss.env');
+
             Zenoss.env.COLLECTORS = %r;
             Zenoss.env.priorities = %r;
             Zenoss.env.productionStates = %r;
             Zenoss.USER_TIMEZONE = "%s" || jstz.determine().name();
-        """ % ( collectors, priorities, productionStates, timezone )
-        return snippet
+            Zenoss.USER_DATE_FORMAT = "%s" || "MM/DD/YY";
+          })();
```

Comment:
@rDr4g0n I've changed this snippet to be in a local scope. It's working exactly as in [zenoss.js](../../tree/develop/Products/ZenUI3/browser/resources/js/zenoss/zenoss.js#L8) (and many other places). I believe it's done properly this time.